cmake/modules/RootMacros.cmake

```diff
@@ -467,11 +469,11 @@ function(ROOT_GENERATE_DICTIONARY dictionary)
 
   #---what rootcling command to use--------------------------
   if(ARG_STAGE1)
-    set(command ${CMAKE_COMMAND} -E env "LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib:$ENV{LD_LIBRARY_PATH}" $<TARGET_FILE:rootcling_stage1>)
+    set(command ${CMAKE_COMMAND} -E env "${ld_library_path}=${CMAKE_BINARY_DIR}/lib:$ENV{${ld_library_path}}" "LSAN_OPTIONS=$ENV{LSAN_OPTIONS}:exitcode=0" $<TARGET_FILE:rootcling_stage1>)
```

Comment:
Can we add here a conditional check if actually we have $ENV{LSAN_OPTIONS} then set command otherwise don't touch it...It will not pollute rootcling_stage1 nd rootcling invocations.

Reply:
Ok, so you want to prevent a set but empty environment variable?